src/lando/api/tests/test_transplants.py

```diff
@@ -962,6 +963,26 @@ def test_integrated_transplant_records_approvers_peers_and_owners(
     assert approved_by == [[101], [102]]
 
     assert hg_landing_worker.run_job(job)
+    assert job.landed_commit_id
+
+    # Fetch Job data.
+    response = authenticated_client.get(
```

Comment:
This tests real HTTP interactions, rather than calling internal methods, so we use the normal test client, rather than the `proxy_client`